docs/components/geometry.md

```diff
@@ -47,9 +82,21 @@ The circle primitive defines two-dimensional circles, which can be complete circ
 | thetaStart  | Start angle for first segment. Can be used to define a partial circle.                                                           | 0             |
 | thetaLength | The central angle (in degrees). Defaults to `360`, which makes for a complete circle.                                            | 360           |
 
-### Cone
+#### `thetaLength` and `thetaStart` Properties
+
+In degrees, `thetaStart` defines where to start a circle or arc and
+`thetaLength` defines where a circle or arc ends. If we wanted to make a `(`
+shape, we would start the circle halfway through and define the length as half
+of a circle. We can do this with `thetaStart: 180; thetaLength: 180`. Or if we
+wanted to make a `)` shape. We can do do `thetaStart: 0; thetaLength: 180`.
+
+Useful cases might be to animating `thetaStart` to create a spinner effect or
+animating `thetaLength` on a fuse-based cursor for visual feedback.
+
+
+### `cone`
 
-The cone primitive under the hood is a cylinder primitive with varying top and bottom radiuses.
+The cone geometry is a cylinder geometry that can varying top and bottom radiuses.
```

Comment:
"that can varying" is a bit awkward.

Suggestion: "The cone geometry is a cylinder geometry with varying top and bottom radii."